sdks/go/pkg/beam/core/runtime/harness/harness.go

```diff
@@ -158,6 +158,60 @@ func Main(ctx context.Context, loggingEndpoint, controlEndpoint string) error {
 type bundleDescriptorID string
 type instructionID string
 
+const circleBufferCap = 1000
+
+// circleBuffer is an ordered eviction buffer
+type circleBuffer struct {
+	// a plan that's either about to start or has finished recently
```

Comment:
nit: This comment doesn't make sense anymore because there's no indication that this map is for inactive plans. Can probably adjust these comments a bit, maybe delete this one and amend the one below to refer to "buf" instead of "inactive map".

Reply:
Ah! good catch. Thanks!